lib/hipchat/api_version.rb

```diff
@@ -302,6 +302,15 @@ def view_config
         }[version]
       end
 
+      def user_room_config
+        {
+          'v2' => {
```

Comment:
Ahh, I missed that I was in a hurry, since I gotta use this part. Let me do this.